raspberry/bzzz/read_sbus/radioDataParser.py

```diff
@@ -57,21 +57,26 @@ def __init__(self):
         self.rawRatePercentage = None
         self.switchValue = None
 
-    def __map_radio_to_angle(self, x):
+    def __radio_calib(self):
+        """Takes 
+        """
+        pass
+
+    def __map_radio_to_angle(self, x, stick_min=RADIO_STICK_MIN, stick_max=RADIO_STICK_MAX):
         """Maps radio stick positions to corresponding angles in radians linearly.
 
         :param x: Stick position data.
         :return: angle in radians.
```

Comment:
Add `stick_min` and `stick_max` to the docs

Reply:
It is documented in line 6.

Reply:
This is the documentation I see for this function:

```
"""Maps radio stick positions to corresponding angles in radians linearly.
        :param x: Stick position data.
        :return: angle in radians.
"""
```

Strangely, there are some more docstring-like comments in lines 74-78.

Reply:
I see. Sorry, I was looking at a different line. I am on it.